### ISSUE TITLE
Fix FlatList not focusing on tap with TalkBack in category list items

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#4360]
### Describe the Bug

Currently we are using nested FlatList in category list items which is a bad practice as it leads to `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead.` if `scrollEnabled` is not set to `false`. Seems like the fix with `scrollEnabled={false}` just hides the error warning and does not solves it.
`scrollEnabled={false}` should be avoided to not break the tap behaviour when TalkBack is activated (could be also the case with VoiceOver in ios)

### Steps to Reproduce

1. Activate TalkBack on android 
2. Go to any category
3. Try to tap on the category list items
4. See that it is not focusing on the tapped list item

### Environment

- Device/Browser: N/A
- OS version: N/A
- App version: N/A

### Related Issues

could be relatable #4080 

### Additional Information

N/A

**Comment by bahaaTuffaha** [comment]
Couldn't reproduce the problem.

**Comment by lunars97** [comment]
> Couldn't reproduce the problem.

hmm strange... I will have a look

**Comment by lunars97** [comment]
> Couldn't reproduce the problem.

I was able to reproduce it on main, when you go to category -> tap on any category item in a list -> it is not being tapped and green outline is not appearing at all... See the recording:

https://github.com/user-attachments/assets/fb1afe5e-73da-48d0-ad0f-faf2cf69ba8b
